MODULE.bazel

```diff
@@ -51,6 +51,15 @@ bazel_dep(name = "rules_go", version = "0.52.0")
 bazel_dep(name = "protobuf", version = "27.3", dev_dependency = True)
 bazel_dep(name = "toolchains_protoc", version = "0.3.2", dev_dependency = True)
 
+# make sure no .cc file gets compiled for golang protobuf support
```

Comment:
this is also in the bazel 8 PR

Reply:
Yeah, I took it from you and improved before telling you how to make it better